src/components/LandingHowItWorks/index.tsx

```diff
@@ -120,7 +120,7 @@ const columnsData = [
   {
     title: "Sensoren",
     description:
-      "Wir unterstützen aktuell die folgenden, für die Umwelt relevanten Sensortypen.",
+      "Prinzipiell kannst du jeden Sensor verbinden. Wir unterstützen aktuell die folgenden, für die Umwelt relevanten, Sensortypen und machen sie mit einem Kategorie-Flag sichtbar.",
```

Comment:
Besser IMHO: "...Sensortypen und machen sie mit eine~~m~~ Kategorie~~Flag~~ sichtbar

Reply:
good point --> will add it

Reply:
Changed this already!